### PR TITLE
Updates resources and goals api to follow api.apiId new name convention

### PR DESCRIPTION
**Pull request by TylerHernandez**

### CODE REVIEW

**Review by Mike-Volchko — APPROVED** [review]
Looks good.

File: client/agora/public/assets/uploads/resource/notebook-pen.svg [hunk 1]
```diff
@@ -1 +0,0 @@
...s><style>.cls-1{fill-rule:evenodd;}</style></defs><title>notebook</title><path class="cls-1" d="M103.3,34.19l8.23,3.52a2.15,2.15,0,0,1,1.13,2.82l-2,4.56L98.53,39.88l2-4.56a2.15,2.15,0,0,1,2.82-1.13ZM8.88,7.88h8.19V2.73a2.74,2.74,0,0,1,5.47,0V7.88h12V2.73a2.73,2.73,0,1,1,5.46,0V7.88H52V2.73a2.73,2.73,0,0,1,5.46,0V7.88h12V2.73a2.73,2.73,0,0,1,5.46,0V7.88h9.27a8.91,8.91,0,0,1,8.88,8.88V28.54a12.27,12.27,0,0,0-1.76,2.9l-2,4.56A10,10,0,0,0,89,37.16a11.24,11.24,0,0,0-.58,1.15l-.6,1.4V16.76a3.6,3.6,0,0,0-3.58-3.58H75v5.15a2.73,2.73,0,0,1-5.46,0V13.18h-12v5.15a2.73,2.73,0,0,1-5.46,0V13.18H40v5.15a2.73,2.73,0,1,1-5.46,0V13.18h-12v5.15a2.74,2.74,0,0,1-5.47,0V13.18H8.88A3.58,3.58,0,0,0,5.3,16.76v92a3.6,3.6,0,0,0,3.58,3.59H59.16l.56,5.29H8.88A8.89,8.89,0,0,1,0,108.77v-92A8.91,8.91,0,0,1,8.88,7.88ZM20.34,94.35a2.65,2.65,0,0,1,0-5.3H66.72l-2.27,5.3Zm0-17.48a2.65,2.65,0,0,1,0-5.3H72.78a2.52,2.52,0,0,1,1.27.35l-2.12,5Zm0-17.48a2.65,2.65,0,0,1,0-5.3H72.78a2.65,2.65,0,0,1,0,5.3Zm0-17.48a2.65,2.65,0,0,1,0-5.3H72.78a2.65,2.65,0,0,1,0,5.3ZM81,114.6l-6.19,5c-4.85,3.92-4.36,5.06-5-.88l-1-9.34h0L97.54,42.18l12.18,5.22L81,114.6Zm-10.09-4.31,8,3.42L74.82,117c-3.19,2.58-2.87,3.32-3.28-.57l-.66-6.14Z"/></svg>
```
Comment (cmarcusdimarco) [inline client/agora/public/assets/uploads/resource/notebook-pen.svg:1]: This image missing in the PR makes me think we didn't fully resolve the saveImage issue. @Mike-Volchko we should look into this.

**Review by cmarcusdimarco — CHANGES_REQUESTED** [review]
I am all set to approve once notebook-pen.svg is restored.

**Comment by cmarcusdimarco** [comment]
@briangormanly ready when you are on this one - don't want to push it through without your review.

**Review by briangormanly — APPROVED** [review]
good to god.

**Pull request merged**